DOC-2016: Added TINY-9749 to the release notes. (#2712)
tinymce/tinymce-docs: modules/ROOT/pages/6.5-release-notes.adoc

The following premium plugin updates were released alongside {productname} 6.5.

=== Advanced Code Editor 3.3.0

The {productname} 6.5 release includes an accompanying release of the **Advanced Code Editor** premium plugin.

**Advanced Code Editor** 3.3.0 includes the following change.

=== Advanced Code Editor did not preserve either the `font size` and `dark mode` save state when closed.
//#TINY-9749

In previous versions of {productname}, the `font size` and `dark mode` state within the Advanced Code Editor was not preserved after closing the dialog.

As a consequence, when the user closed the Advanced Code Editor the editor would reset the state back to its default settings.

In {productname} 6.5, the state for the `font size` and `dark mode` status is now preserved when the Advanced Code Editor is closed.

> 

Author: ```suggestion

[NOTE]
This issue only affected the Advanced Coded Editor when it was is in modal.
```

The following premium plugin updates were released alongside {productname} 6.5.

=== Advanced Code Editor 3.3.0

The {productname} 6.5 release includes an accompanying release of the **Advanced Code Editor** premium plugin.

**Advanced Code Editor** 3.3.0 includes the following change.

=== Advanced Code Editor did not preserve either the `font size` and `dark mode` save state when closed.
//#TINY-9749

In previous versions of {productname}, the `font size` and `dark mode` state within the Advanced Code Editor was not preserved after closing the dialog.

As a consequence, when the user closed the Advanced Code Editor the editor would reset the state back to its default settings.

In {productname} 6.5, the state for the `font size` and `dark mode` status is now preserved when the Advanced Code Editor is closed.

[NOTE]

> This issue only affected the Advanced Coded Editor when it was is in modal.
> [[accompanying-premium-plugin-end-of-life-announcement]]

Author: ```suggestion
This issue only affected the Advanced Coded Editor when it was is in modal.

[[accompanying-premium-plugin-end-of-life-announcement]]
```

The following premium plugin updates were released alongside {productname} 6.5.

> === Advanced Code Editor 3.3.0
> 
> The {productname} 6.5 release includes an accompanying release of the **Advanced Code Editor** premium plugin.
> 
> **Advanced Code Editor** 3.3.0 includes the following change.
> 
> === Advanced Code Editor did not preserve either the `font size` and `dark mode` save state when closed.
> //#TINY-9749
> 
> In previous versions of {productname}, the `font size` and `dark mode` state within the Advanced Code Editor was not preserved after closing the dialog.
> 
> As a consequence, when the user closed the Advanced Code Editor the editor would reset the state back to its default settings.
> 
> In {productname} 6.5, the state for the `font size` and `dark mode` status is now preserved when the Advanced Code Editor is closed.
> 
> [NOTE]
> This issue only affected the Advanced Coded Editor when it was is in modal.

Reviewer: ```suggestion
=== Advanced Code Editor 3.3.0

The {productname} 6.5 release includes an accompanying release of the **Advanced Code Editor** premium plugin.

**Advanced Code Editor** 3.3.0 includes the following change.

=== Advanced Code Editor did not preserve changes to _font size_ or _dark/light mode_ state.
//#TINY-9749

In previous versions of {productname}, the _font size_ and _dark/light mode_ states within the Advanced Code Editor were not preserved after closing the dialog.

When the Advanced Code Editor was closed, these states reverted to their default values.

In {productname} 6.5, _font size_ and _dark/light mode_ states are preserved when the Advanced Code Editor is closed.

NOTE: This issue only affected the Advanced Coded Editor when it was opened in a dialog.

For information on the **Advanced Code Editor** plugin, see: xref:advcode.adoc[Advanced Code Editor].
```
Edits.

The following premium plugin updates were released alongside {productname} 6.5.

=== Advanced Code Editor 3.3.0

The {productname} 6.5 release includes an accompanying release of the **Advanced Code Editor** premium plugin.

**Advanced Code Editor** 3.3.0 includes the following change.

=== Advanced Code Editor did not preserve changes to _font size_ or _dark/light mode_ state.
//#TINY-9749

> In previous versions of {productname}, the _font size_ and _dark/light mode_ states within the Advanced Code Editor were not preserved after closing the dialog.

Reviewer: ```suggestion
In previous versions of the Advanced Code Editor, the _font size_ and _dark/light mode_ states within the Advanced Code Editor were not preserved after closing the dialog.
```

> In previous versions of the Advanced Code Editor, the _font size_ and _dark/light mode_ states within the Advanced Code Editor were not preserved after closing the dialog.
> 
> When the Advanced Code Editor was closed, these states reverted to their default values.
> 
> In {productname} 6.5, _font size_ and _dark/light mode_ states are preserved when the Advanced Code Editor is closed.
> 
> NOTE: This issue only affected the Advanced Coded Editor when it was opened in a dialog.

Reviewer: ```suggestion
In previous versions of the **Advanced Code Editor**, the _font size_ and _dark/light mode_ states were not preserved after closing the dialog.

When the **Advanced Code Editor** was closed, these states reverted to their default values.

In **Advanced Code Editor** 3.3.0, _font size_ and _dark/light mode_ states are preserved when the editor is closed.

NOTE: This issue only affected the **Advanced Coded Editor** when it was opened in a dialog.
```
